swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
I see no reason to embed swagger ui in the REST server. It made loads of sense for APIMan which was all about managing APIs. This is not. We should document the swagger.json URL and recommend stuff like Swagger Chrome extension (https://chrome.google.com/webstore/detail/swagger-ui-console/ljlmonadebogfjabhkppkoohjkjclfai).